renovate.json5

```
@@ -0,0 +1,1275 @@
+{
```

Should we put something like this:
```suggestion
// --- This file is automatically generated by scripts/renovate-config-generator.mjs, do not edit manually! ---
{
```